ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.js

```diff
@@ -40,17 +43,23 @@ const EditGasFeePopover = () => {
                 <I18nValue messageKey="gasOption" />
               </span>
               <span className="edit-gas-fee-popover__content__header-time">
-                <I18nValue messageKey="time" />
+                {editGasMode !== EDIT_GAS_MODES.SWAPS && (
+                  <I18nValue messageKey="time" />
+                )}
               </span>
               <span className="edit-gas-fee-popover__content__header-max-fee">
                 <I18nValue messageKey="maxFee" />
               </span>
             </div>
-            <EditGasItem priorityLevel={PRIORITY_LEVELS.LOW} />
+            {editGasMode !== EDIT_GAS_MODES.SWAPS && (
+              <EditGasItem priorityLevel={PRIORITY_LEVELS.LOW} />
+            )}
             <EditGasItem priorityLevel={PRIORITY_LEVELS.MEDIUM} />
             <EditGasItem priorityLevel={PRIORITY_LEVELS.HIGH} />
```

Comment:
<img width="403" alt="Screen Shot 2021-12-08 at 3 27 51 PM" src="https://user-images.githubusercontent.com/34557516/145287009-788e35c6-aa4f-427d-9560-261aeeddb7d1.png">
are we supposed to see the high/aggressive option?

Reply:
Ok I see [here](https://github.com/MetaMask/metamask-extension/pull/12966/files#diff-1a63f62a59b5000695c3aa2612b419287852e21c9dc2857e12c166ad75b99168R120) the test states we shouldn't. Why aren't we doing this filtering at this level?

Reply:
aggressive = swap suggested for swaps, it works same just different label

Reply:
I feel like it might be more clear to just have a `<EditGasItem priorityLevel={PRIORITY_LEVELS.SWAPS} />` here? But wouldn't block on it!